pkg/controller/kogitoinfra/infinispan.go

```diff
@@ -61,36 +73,42 @@ var (
 
 	// propertiesInfinispanQuarkus infinispan properties for quarkus runtime
 	propertiesInfinispanQuarkus = map[int]string{
-		appPropInfinispanServerList:    "quarkus.infinispan-client.server-list",
-		appPropInfinispanUseAuth:       "quarkus.infinispan-client.use-auth",
-		appPropInfinispanSaslMechanism: "quarkus.infinispan-client.sasl-mechanism",
-		appPropInfinispanAuthRealm:     "quarkus.infinispan-client.auth-realm",
+		appPropInfinispanServerList:         "quarkus.infinispan-client.server-list",
+		appPropInfinispanUseAuth:            "quarkus.infinispan-client.use-auth",
+		appPropInfinispanSaslMechanism:      "quarkus.infinispan-client.sasl-mechanism",
+		appPropInfinispanAuthRealm:          "quarkus.infinispan-client.auth-realm",
+		appPropInfinispanTrustStore:         "quarkus.infinispan-client.trust-store",
+		appPropInfinispanTrustStoreType:     "quarkus.infinispan-client.trust-store-type",
+		appPropInfinispanTrustStorePassword: "quarkus.infinispan-client.trust-store-password",
 
 		envVarInfinispanUser:     "QUARKUS_INFINISPAN_CLIENT_AUTH_USERNAME",
 		envVarInfinispanPassword: "QUARKUS_INFINISPAN_CLIENT_AUTH_PASSWORD",
 	}
 	// propertiesInfinispanSpring infinispan properties for spring boot runtime
 	propertiesInfinispanSpring = map[int]string{
-		appPropInfinispanServerList:    "infinispan.remote.server-list",
-		appPropInfinispanUseAuth:       "infinispan.remote.use-auth",
-		appPropInfinispanSaslMechanism: "infinispan.remote.sasl-mechanism",
-		appPropInfinispanAuthRealm:     "infinispan.remote.auth-realm",
+		appPropInfinispanServerList:         "infinispan.remote.server-list",
+		appPropInfinispanUseAuth:            "infinispan.remote.use-auth",
+		appPropInfinispanSaslMechanism:      "infinispan.remote.sasl-mechanism",
+		appPropInfinispanAuthRealm:          "infinispan.remote.auth-realm",
+		appPropInfinispanTrustStore:         "infinispan.remote.trust-store-path",
```

Comment:
```suggestion
		appPropInfinispanTrustStore:         "infinispan.remote.trust_store_file_name",
```
trust-store-path is used just for PEM format, see https://docs.jboss.org/infinispan/11.0/apidocs/org/infinispan/client/hotrod/configuration/package-summary.html

Reply:
Using `trust-store-path` leads to crash of SpringBoot application

Reply:
Sorry was late and I was tired :sweat_smile: